### PR TITLE
add test to isKubeadmConfigPresent method

### PR DESCRIPTION
**Pull request by muyangren2**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Add UT
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
"NONE"
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: cmd/kubeadm/app/cmd/upgrade/common_test.go [hunk 1]
```diff
@@ -151,3 +153,38 @@ func TestPrintConfiguration(t *testing.T) {
...
+			name: "Empty Group value",
+			gvkmap: kubeadmapi.DocumentMap{
+				{Group: "", Version: "v1", Kind: "Empty"}: []byte(`kind: Empty`),
+			},
+			expected: false,
+		},
+		{
+			name: "Correct Group value",
+			gvkmap: kubeadmapi.DocumentMap{
+				{Group: "kubeadm.k8s.io", Version: "v1", Kind: "Empty"}: []byte(`kind: Empty`),
```
Comment (yangjunmyfm192085) [inline cmd/kubeadm/app/cmd/upgrade/common_test.go:180]: Wouldn't it be better to use `kubeadmapi.GroupName`?

File: cmd/kubeadm/app/cmd/upgrade/common_test.go [hunk 2]
```diff
@@ -151,3 +153,38 @@ func TestPrintConfiguration(t *testing.T) {
...
 }
+
+func TestIsKubeadmConfigPresent(t *testing.T) {
+	var tcases = []struct {
+		name     string
+		gvkmap   kubeadmapi.DocumentMap
+		expected bool
+	}{
+		{
+			name: " Wrong Group value",
```
Comment (yangjunmyfm192085) [inline cmd/kubeadm/app/cmd/upgrade/common_test.go:164]: Will the gvk.Group in the docmap be nil?

**Review by yangjunmyfm192085 — COMMENTED** [review]
/lgtm

**Review by neolit123 — COMMENTED** [review]
/approve
thanks

**Pull request merged**